service/grpc/codelingo/codelingo.pb.go

```diff
@@ -346,13 +213,6 @@ func (m *ListLexiconsReply) String() string            { return proto.CompactTex
 func (*ListLexiconsReply) ProtoMessage()               {}
```

Comment:
exported method ListLexiconsReply.Descriptor should have comment or be unexported